cime_config/machines/config_machines.xml

```diff
@@ -179,7 +179,7 @@
         <command name="rm">git</command>
         <command name="load">git</command>
         <command name="rm">cmake</command>
-        <command name="load">cmake/3.20.2</command>
+        <command name="load">cmake/3.21.3</command>
```

Comment:
The new version of cmake in config_machines.xml is needed to run on Cori.  This change was discussed on Slack, so I am surprised it hasn't already been merged to next/master.

Reply:
@cameronsmith1 , thanks for the suggestion.

@rljacob , I feel this machine config change doesn't belong to this PR. Where and who should implement it?

Reply:
The change to the cmake version has been made as noted by @ndkeen below.